Assets/Scripts/Interactables/Collectable.cs

```diff
@@ -1,14 +1,24 @@
 using Managers;
+using UnityEditor.EditorTools;
```

Comment:
Is this used? `UnityEditor.*` namespaces aren't defined in standalone builds so this breaks Standalone builds

(I learned this the hard way a few years back the day before a deadline lol)

Reply:
unity moment